src/core/raster/qgsrasterdrawer.h

```diff
@@ -41,9 +41,16 @@ class CORE_EXPORT QgsRasterDrawer
     /**
      * The QgsRasterDrawer constructor.
      * \param iterator the raster iterator to fetch data from
-     * \param dpiTarget the target \a dpi (dots per inch) to be taken into consideration when rendering
+     * \param dpiTarget the target dpi (dots per inch) to be taken into consideration when rendering
+     * \deprecated since QGIS 3.28
```

Comment:
```suggestion
     * \deprecated since QGIS 3.28. Use the constructor without the \a dpiTarget argument instead, as DPI is now handled by the draw() method which accepts a QgsRenderContext.
```